tests/test_more.py

```diff
@@ -1915,7 +1915,7 @@ def test_deprecation(self):
             )
 
         (warning,) = caught
-        assert warning.category == DeprecationWarning
+        self.assertTrue(warning.category is DeprecationWarning)
```

Comment:
Consider using [`assertIs`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIs) for this line or simplify the entire test with [`assertWarns`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertWarns).